### PR TITLE
Make CSV use OS standard

### PR DESCRIPTION
**Pull request by nachoiacovino**
CSV upload wasn't using OpenSea standard, and would break OpenSea if the attribute column was empty, this standarizes to use OS standard everywhere and fixes that bug

### CODE REVIEW

File: utils/batch.ts [hunk 1]
```diff
@@ -95,6 +96,21 @@ export const removeEmptyKeysFromObject = (obj: any) => {
...
+  const keys = Object.keys(obj || {});
+  const values = Object.values(obj || {});
+  const attributes = [];
+
+  for (let i = 0; i < keys.length; i++) {
+    attributes.push({
+      trait_type: keys[i],
+      value: values[i],
+    });
+  }
```
Comment (jnsdls) [inline utils/batch.ts:109]: could be replaced with this

```suggestion
  const attributes = Object.entries(obj || {}).map(([trait_type, value]) => ({
    trait_type,
    value,
  }));
```

File: utils/parseAttributes.ts [hunk 2]
```diff
@@ -11,16 +11,16 @@ export function parseAttributes<
...
   if (!data) {
     return data;
   }
   if (Array.isArray(data)) {
     const parsedArray = data
       .filter(({ value }) => value !== "")
       .map(({ value, trait_type }) => ({
-        value,
         ...(!!trait_type && { trait_type }),
+        value,
```
Comment (jnsdls) [inline utils/parseAttributes.ts:23]: why change the order here?

**Review by jnsdls — COMMENTED** [review]
some quick questions / suggestions

File: utils/parseAttributes.ts (hunk 2 above)
Comment (nachoiacovino) [reply utils/parseAttributes.ts:23]: makes more sense in the preview of the batch upload seeing trait_type first

File: utils/batch.ts (hunk 1 above)
Comment (nachoiacovino) [reply utils/batch.ts:109]: well that's miles easier, thanks

File: utils/parseAttributes.ts (hunk 2 above)
Comment (jnsdls) [reply utils/parseAttributes.ts:23]: ah so just visual ordering, ok

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**